UnitTests/Views/TextFieldTests.cs

```diff
@@ -2047,4 +2047,18 @@ public void Autocomplete_Visible_False_By_Default ()
         Assert.True (t.Visible);
         Assert.False (t.Autocomplete.Visible);
     }
+
+    [Fact]
+    [AutoInitShutdown]
```

Comment:
This test does not require use of AutoInitShutdown.

Reply:
It's needed because tf.Draw calls Driver.SetAttribute.